src/Microsoft.Identity.Web.TokenAcquisition/PlainDotNetTokenAcquisitionHost.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.Net;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+using Microsoft.Identity.Client;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Microsoft.Identity.Web.Hosts
+{
+    internal class PlainDotNetTokenAcquisitionHost : ITokenAcquisitionHost
```

Comment:
let's think of a better name

Reply:
DefaultTokenAcquisitionHost seems good indeed.

Reply:
renamed.